### PR TITLE
feat(filters): support autoCapitalize & autoCorrect

### PR DESCRIPTION
**Pull request by glennreyes**
### Description

Added `autoCapitalize` & `autoCorrect` to the whitelist which are non-standard DOM attributes for Mobile Safari. These are [supported in React](https://facebook.github.io/react/docs/dom-elements.html#all-supported-html-attributes).
Added a `yarn.lock` file to improve packaging.

### Checklist

- [x] All tests passing
- [x] Created tests which fail without the change (if possible)
- [x] Extended the README / documentation, if necessary
- [x] Commits and PR follow conventions

### CODE REVIEW

**Comment by joanrodriguez** [comment]
Great one

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-geosuggest",
-  "version": "2.4.0",
+  "version": "2.4.1",
```
Comment (ro-ka) [inline package.json]: Please do not release in feature branches! The release will be done in the master branch and may have more than one PR in it.

File: src/filter-input-attributes.js [hunk 2]
```diff
@@ -2,8 +2,10 @@
  * Attributes allowed on input elements
  */
 const allowedAttributes = [
-  'autoFocus',
+  'autoCapitalize',
   'autoComplete',
+  'autoCorrect',
+  'autoFocus',
```
Comment (ro-ka) [inline src/filter-input-attributes.js]: 👍

**Comment by glennreyes** [comment]
Thanks for feedback please revisit!

**Comment by ro-ka** [comment]
Thanks for the fast update!

**Comment by ro-ka** [comment]
Released in `2.5.0`. Thanks!

**Comment by glennreyes** [comment]
Perfect, thanks1

**Pull request merged**